doc/install/fast-install-and-first-run.rst

```diff
@@ -94,7 +112,58 @@ Database init, data download & importation
    $ sudo ivre ipdata --download --import-all
 
 The two last step may take a long time to run (40 minutes on a decent
-server), nothing to worry about.
+server), nothing to worry about. To initiate flowcli, Neo4j default 
```

Comment:
This is going to be unnecessary very soon (#685).

Reply:
Added a comment below.

Reply:
Same, #685 has been merged.

Reply:
Okay